dbms/src/Server/Server.cpp

```diff
@@ -1096,6 +1096,16 @@ int Server::main(const std::vector<std::string> & /*args*/)
     /// Initialize RateLimiter.
     global_context->initializeRateLimiter(config(), bg_pool, blockable_bg_pool);
 
+    global_context->setServerInfo(server_info);
```

Comment:
`ServerInfo` depends on proxy, but the TiFlash compute node will not have the proxy, so the double value will not work in TiFlash compute node, is it the expected behavior?

Reply:
Yes. IMHO, even though compute node will not have the proxy, it can still get memory info from `UInt64 capacity = getMemoryAmount();`. This take effect on my mac and centos7.

And even tough they all failed to get total RAM, we will print a ERROR log to remind user don't use double value.